sources/PTYSession.m

```diff
@@ -7556,6 +7558,20 @@ - (void)mitigateTouchBarStupidityForEvent:(NSEvent *)event {
     [self showVisualIndicatorForEvent:event];
 }
 
+- (void)textViewSelectionDidChangeToTruncatedString:(NSString *)selection {
+    // Assign a maximum of maximumBytesToProvideToPythonAPI characters to the "selection"
+    // iTerm Variable
+    //
+    // The "selectionLength" iTerm variable contains the full length of the original
+    // selection; not the restricted length assigned to the "selection" iTerm Variable
+    DLog(@"textViewSelectionDidChangeToTruncatedString: %@", selection);
+
+    selection = selection ? selection : @"";
```

Comment:
Don’t assign to an argument. Rename the argument to `maybeSelection` and then do `NSString *selection = maybeSelection ?: @""`.

Reply:
Done.  I didn't know of the `<cond> ?: <fallback>` syntax - thanks.